recipes/wt/all/conanfile.py

```diff
@@ -0,0 +1,204 @@
+        "with_sqlite": [True, False],
+        "with_postgres": [True, False],
+        "with_firebird": [True, False],
+        "with_mysql": [True, False],
+        "with_mssql": [True, False],
+        "with_qt4": [True, False],
+        "with_test": [True, False],
+        "with_dbo": [True, False],
+        "with_opengl": [True, False],
+        "with_unwind": [True, False],
+        "no_std_locale": [True, False],
+        "no_std_wstring": [True, False],
+        "multi_threaded": [True, False],
+        "connector_http": [True, False],
+        "connector_isapi": [True, False],
+        "connector_fcgi": [True, False]
+    }
+    default_options = {
+        'shared': False,
+        'fPIC': True,
+        'with_ssl': True,
+        'with_haru': False,
+        'with_pango': False,
+        'with_sqlite': True,
+        'with_postgres': True,
+        'with_firebird': False,
+        'with_mysql': True,
+        'with_mssql': False,
+        'with_qt4': False,
+        'with_test': True,
+        'with_dbo': True,
+        'with_opengl': False,
+        'with_unwind': True,
+        'no_std_locale': False,
+        'no_std_wstring': False,
+        'multi_threaded': True,
+        'connector_http': True,
+        'connector_isapi': True,
+        'connector_fcgi': False
+        }
+
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+
+    _cmake = None
+
+    requires = ('zlib/1.2.11', 'boost/1.73.0')
+
+    def requirements(self):
+        if self.options.with_ssl:
+            self.requires('openssl/1.1.1g')
+        if self.options.with_sqlite:
+            self.requires('sqlite3/3.31.1')
+        if self.options.with_mysql:
+            self.requires('libmysqlclient/8.0.17')
+        if self.options.with_postgres:
+            self.requires('libpq/11.5')
+        if self.options.with_unwind:
+            self.requires('libunwind/1.3.1')
+
+    def config_options(self):
+        if self.settings.os == 'Windows':
+            del self.options.fPIC
+            del self.options.connector_fcgi
+        else:
+            del self.options.connector_isapi
+        if self.settings.os not in ["Linux", "FreeBSD"]:
+            self.options.with_unwind = False
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions['CONFIGDIR'] = os.path.join(self.package_folder, 'bin')
+        self._cmake.definitions['SHARED_LIBS'] = self.options.shared
+        self._cmake.definitions['BUILD_EXAMPLES'] = False
+        self._cmake.definitions['BUILD_TESTS'] = False
+        self._cmake.definitions['ENABLE_SSL'] = self.options.with_ssl
+        self._cmake.definitions['ENABLE_HARU'] = self.options.with_haru
```

Comment:
haru, pango, firebird and qt, we don't packages for them. I don't know if we should enable them from the system, as you know, it's hard to track any system version. I would prefer them until having available;

Reply:
So you want me to remove these 4 options, and always disable the corresponding part, Do I understand correctly ?

Reply:
That's one option. Another one is raising ConanInvalidConfiguration when enabled.